src/test/java/uk/gov/companieshouse/extensions/api/reasons/ReasonServiceUnitTest.java

```diff
@@ -104,10 +101,8 @@ public void canGetListOfReasons() throws ServiceException {
 
     @Test
     public void willThrowIfNoRequestExists() throws ServiceException {
-        exception.expect(ServiceException.class);
-        exception.expectMessage("Extension request 123 not found");
 
-        reasonsService.getReasons("123");
+        assertThrows(ServiceException.class, () -> reasonsService.getReasons("123"));
```

Comment:
we've lost the message check here for "Extension request 123 not found"

Reply:
Added a check.